org.eclipse.xtend.ide/META-INF/MANIFEST.MF

```diff
@@ -15,15 +15,15 @@ Require-Bundle: org.eclipse.xtend.core;bundle-version="2.37.0",
  org.eclipse.xtext.xbase.ui;bundle-version="2.37.0",
  org.eclipse.xtend.ide.common;bundle-version="2.37.0";visibility:=reexport,
  org.eclipse.ui;bundle-version="3.201.0",
- org.eclipse.jdt.core;bundle-version="3.29.0",
+ org.eclipse.jdt.core;bundle-version="3.37.0",
  org.eclipse.ui.ide;bundle-version="3.13.1",
  org.eclipse.ui.console;bundle-version="3.11.100",
- org.eclipse.core.filesystem;bundle-version="1.9.300",
+ org.eclipse.core.filesystem;bundle-version="1.10.300",
  org.eclipse.jdt.ui;bundle-version="3.26.0",
```

Comment:
We should try to be more consistent with the require bundle dependencies, e.g. jdt.ui 3.26 is unlikely to match jdt.core 3.37

Reply:
@szarnekow I only updated the dependencies when there was a matching one in the dev BOM, IIRC.

In general, I'm OK with keeping in sync minimal versions for the same dependencies that we have in the BOM.
For the other ones, I'd like to avoid such a massive maintenance burden...